### ISSUE TITLE
Test for WCAG failure F8: captions omitting some dialogue or important sound effects

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#152]
http://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/F8

This describes a failure condition for all techniques involving captions. If the "caption" does not include all of the dialogue (either verbatim or in essence) as well as all important sounds then the 'Captions' are not real captions. 

Examples of text streams that are not captions include:
-  text that contains the dialogue (possibly simplified dialogue) but that does not describe important sounds
-  text that omits dialogue during portions of the material 
## Procedure
1. View the material with captioning turned on.
2. Check that all dialogue is accompanied by a caption.
3. Check that all important sounds are captioned.
